src/acronyms.ts

```diff
@@ -88,9 +430,20 @@ export function registerAcronymListeners(client: Client): void {
     try {
       if (message.author.bot) return;
 
-      if (message.channel.id !== WATERCOOLER_CHANNEL_ID) return;
+      let acronyms: AcronymMap;
+      let detectedAcronyms: string[] = [];
```

Comment:
Don't need to forward-declare this, can (and should) just declare this as a const on line 446 since it never gets redefined.

Reply:
woooops

clearly i've been spoiled by the linting at Motivo which woulda caught this in-IDE

Fixed! Thanks!

Also TIL that this pattern is called `forward-declaring`